packages/haproxy/data_stream/log/elasticsearch/ingest_pipeline/default.yml

```
@@ -157,11 +157,11 @@ processors:
   - append:
       field: event.category
       value: network
-      if: "ctx?.source.ip != null && ctx?.destination?.ip != null"
+      if: "ctx?.source?.ip != null && ctx?.destination?.ip != null"
   - append:
       field: event.type
       value: connection
-      if: "ctx?.source.ip != null && ctx?.destination?.ip != null"
+      if: "ctx?.source?.ip != null && ctx?.destination?.ip != null"
```

```suggestion
      if: ctx.source?.ip != null && ctx.destination?.ip != null
```

`ctx` can never be null in this context so we can remove the null-safe operators.